default-filters.ts

```diff
@@ -249,28 +336,47 @@ export const defaultFilters: Record<string, Filter> = {
     },
   },
   'IED: All - Complete': {
-    ...BASE_FILTER,
     description: 'Compare all IED in all namespaces',
     ourSelector: 'IED',
+    theirSelector: '',
+    selectors: {
+      inclusive: false,
+      vals: [],
+      except: [],
+    },
+    attributes: {
+      inclusive: false,
+      vals: [],
+      except: [],
+    },
+    namespaces: {
+      inclusive: false,
+      vals: [],
+      except: [],
+    },
   },
   'IED: All - Only SCL namespace without DataTypes or Privates': {
-    ...BASE_FILTER,
     description:
       'Compare all IEDs, only in the SCL namespace and without referencing DataTypes or look into Private elements',
     ourSelector: 'IED',
+    theirSelector: '',
     selectors: {
       inclusive: false,
       vals: ['Private'],
       except: [],
     },
+    attributes: {
+      inclusive: false,
+      vals: [],
+      except: [],
+    },
     namespaces: {
       inclusive: true,
       vals: ['http://www.iec.ch/61850/2003/SCL'],
       except: [],
     },
   },
   'IED: Compare Two  IEDs': {
-    ...BASE_FILTER,
     description:
       'Allow comparison of two IEDs in the same or a different file (requires user modification)',
     ourSelector: 'IED[name="XAT_BusA_P1"] *',
```

Comment:
I think we need to look at this as well, to compare the same IED in the same file I think our selector has to be a single item but I'm not sure this works at the moment.